doc/networking.md

```diff
@@ -1173,33 +1173,33 @@ have changed type to *random*.
 To be able to route (static or dynamic) on the interface it is
 required to enable forwarding. This setting controls if packets
 received on this interface can be forwarded.
-   ```
-   admin@example:/config/> edit interface eth0
-   admin@example:/config/interface/eth0/> set ipv4 forwarding
-   admin@example:/config/interface/eth0/> leave
-   admin@example:/>
-   ```
+
+```
+admin@example:/config/> edit interface eth0
+admin@example:/config/interface/eth0/> set ipv4 forwarding
+admin@example:/config/interface/eth0/> leave
+admin@example:/>
+```
 
 
 ### IPv6 forwarding
 
-This flag behaves totally different than for IPv4. For IPv6 the
-ability to route between interfaces is always enabled, instead this
-flag controls if the interface will be in host/router mode.
+This flag behaves slightly differently compared to IPv4 forwarding.  It
+controls if the interface will be in host or router mode.
```

Comment:
Should we add something like the following?

> Because of how Linux manages IPv6 forwarding, either enabling or disabling it globally, Infix can not fully control it per interface via this setting. Instead, IPv6 forwarding will be globally enabled whenever at least one interface enables it, otherwise it is disabled.

Reply:
Looks good, much better than what we have now. Thanks, will update!